test/Generator/TestAsset/TestEnum.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace LaminasTest\Code\Generator\TestAsset;
+
+enum TestEnum
```

Comment:
Ah, enums were introduced in 8.1, hence the parser failure :thinking:

Reply:
Ok, yeah, I'm gonna pull PHP 8.0 support, but gonna work on the other issues in the milestone first.